### PR TITLE
ISPN-7228 Creating wrong schema doesn't show error message (addendum)

### PR DESCRIPTION
**Pull request by vblagoje**
Master only. Patch for reopened ISPN-7228. Please review @ryanemerson

### CODE REVIEW

File: src/services/schemas/SchemasService.ts [hunk 1]
```diff
@@ -67,6 +67,7 @@ export class SchemaService {
         error => deferred.reject(error))
       .then(error => {
           if (isNotNullOrUndefined(error)) {
+            this.unregisterProtoSchema(container, schema.fileName);
```
Comment (ryanemerson) [inline src/services/schemas/SchemasService.ts]: How important is this action? Do we need to take action if it fails?

Comment (vblagoje) [inline src/services/schemas/SchemasService.ts]: We should take it only if we get an error @ryanemerson See reopened ISPN-7228 for details

Comment (ryanemerson) [inline src/services/schemas/SchemasService.ts]: So at present this is an async action that we fire and forget, no action is taken if the unregister fails/timesout. Do we want to retry if it fails? Or at least notify the user that the schema was created, but with errors and it hasn't been possible to undo its creation

Comment (vblagoje) [inline src/services/schemas/SchemasService.ts]: I don't think we need to retry. How many times should we retry? The failure of not being able to remove schema is not a big deal @ryanemerson

Comment (vblagoje) [inline src/services/schemas/SchemasService.ts]: @ryanemerson he/she can always remove it!

Comment (ryanemerson) [inline src/services/schemas/SchemasService.ts]: @vblagoje Ok cool, integrating

**Pull request merged**